docs/_static/css/custom.css

```diff
@@ -90,4 +91,161 @@ span.std.std-ref {
 a.reference.external {
     color: #126782;
     font-weight: bold;
-}
\ No newline at end of file
+}
```

Comment:
Could we try to encapsulate the common code among all tags in a div or something? just to remove duplicate code. I don't know if the suggestion they make here will work https://stackoverflow.com/questions/3488807/avoiding-duplicate-styles-in-css

Reply:
I will check this! Thank you :)

Reply:
I tried doing this, but doesn't seem to work for sphinx tags

Reply:
I used this to reduce the common code. Thanks again!